src/discovery/kubernetes/Akka.Discovery.KubernetesApi/Akka.Discovery.KubernetesApi.csproj

```diff
@@ -10,6 +10,7 @@
 
     <ItemGroup>
         <PackageReference Include="Akka.Discovery" Version="$(AkkaVersion)" />
+        <PackageReference Include="Akka.Hosting" Version="0.4.2" />
```

Comment:
We should probably reference this from `Directory.Build.props` so all packages get the same version.

Reply:
Done